jarviscli/plugins/converted.py

```diff
@@ -97,7 +97,7 @@ def do_update(self, s):
     location: Updates location.
     system: Updates system.
     """
-    self = self_jarvis
+    self = self._jarvis
```

Comment:
I was going to point this out along with  some other errors.
In line 105 and 106 print_say() is undefined
My  editor shos ConnectionError is undefined as well as CONNECTION_ERROR_MSG